src/viam/robot/client.py

```diff
@@ -738,12 +759,15 @@ async def discover_components(
         queries: List[DiscoveryQuery],
     ) -> List[Discovery]:
         """
-        Get the list of discovered component configurations.
+        Get a list of discovered potential component configurations for webcam cameras, for example listing different supported resolutions.
```

Comment:
Does discovery only work for webcam cameras? If no, this seems to suggest that

Reply:
Yeah it only works for webcam camera re. rand

Reply:
Is there a chance it will work for other components in the future? I don't want to have to be overly specific in the event that we just haven't gotten to implementing it elsewhere just yet but plan to